CONTRIBUTING.md

```diff
@@ -95,6 +104,12 @@ git branch -u upstream/main
 git pull
 ```
 
+
+### Assigning yourself
```

Comment:
do all contributors have the permissions to assign themselves to things? If not could add a sentence about requesting a maintainer to assign them